JamfPro/JamfPro/JamfPro.pq

```diff
@@ -271,751 +117,484 @@ UAPIResource = (baseurl as text, relativepath as text, optional query as record,
+                                "jamfBinaryVersion",
+                                "platform","barcode1",
+                                "barcode2","assetTag",
+                                "remoteManagement",
+                                "supervised",
+                                "mdmCapable",
+                                "reportDate",
+                                "lastContactTime",
+                                "lastCloudBackupDate",
+                                "lastEnrolledDate",
+                                "mdmProfileExpiration",
+                                "initialEntryDate",
+                                "distributionPoint",
+                                "site",
+                                "itunesStoreAccountActive",
+                                "enrolledViaAutomatedDeviceEnrollment",
+                                "userApprovedMdm",
+                                "enrollmentMethod"
+                            }),
+        expand_disk_encryption = Table.ExpandRecordColumn(expand_general, "diskEncryption", {"individualRecoveryKeyValidityStatus","institutionalRecoveryKeyPresent","diskEncryptionConfigurationName","fileVault2EligibilityMessage","fileVault2EnabledUserNames","bootPartitionEncryptionDetails"}),
+        expand_purchasing = Table.ExpandRecordColumn(expand_disk_encryption, "purchasing", {"purchased","leased","poNumber","lifeExpectancy","purchasePrice","purchasingAccount","purchasingContact","appleCareId","vendor","leaseDate","poDate","warrantyDate"}),
+        expand_user_and_location = Table.ExpandRecordColumn(expand_purchasing, "userAndLocation", {"username","realname","email","position","phone","departmentId","buildingId","room"}),
+        expand_hardware = Table.ExpandRecordColumn(expand_user_and_location, "hardware", {"make","model","modelIdentifier","serialNumber","processorSpeedMhz","processorCount","coreCount","processorType","processorArchitecture","busSpeedMhz","cacheSizeKilobytes","networkAdapterType","macAddress","altNetworkAdapterType","altMacAddress","totalRamMegabytes","openRamSlots","batteryCapacityPercent","smcVersion","nicSpeed","opticalDrive","bootRom","bleCapable","supportsIosAppInstalls","appleSilicon"}),
+        expand_operating_system = Table.ExpandRecordColumn(expand_hardware, "operatingSystem", {"name","version","build","activeDirectoryStatus","fileVault2Status","softwareUpdateDeviceId"}, {"os name","os version","os build","activeDirectoryStatus","fileVault2Status","softwareUpdateDeviceId"})
     in
-        expand_profiles2;
+        expand_operating_system;
 
-computerAttachmentsImpl = (baseUrl as text, pages_table as table) as table =>
+computerAttachmentsImplementation = (baseurl as text, pages_table as table) as table =>
     let
         sections = List.Transform({
             "Attachments"
         }, Text.Upper),
         queryString = Text.Combine(List.Transform(sections, Text.Upper), "&section="),

```

Comment:
I _think_ we can get rid of the unused `queryString` declarations throughout. I imagine they were a holdover from a half-completed refactor.